girder/girder_large_image/web_client/views/imageViewerSelectWidget.js

```diff
@@ -65,6 +66,18 @@ var ImageViewerSelectWidget = View.extend({
         return this;
     },
 
+    _createVueModal(imageMetadata, frameUpdate) {
```

Comment:
Why is this called `_createVueModal` when it doesn't create a modal dialog?  Perhaps it should be renamed to `_createVue`?

Reply:
Sure. This was a remnant I copied over from the previous work. Changed in [e19b401](https://github.com/girder/large_image/pull/1086/commits/e19b401793b7e963a18dfdc8e843041fde15cd67)